spec/game_spec.rb

```diff
@@ -0,0 +1,80 @@
+# rubocop:disable Layout/LineLength
+require './lib/board.rb'
+require './lib/logic.rb'
```

Comment:
- [ ] There is no such directory (/lib/logic.rb). Please check and fix the require statements and if the required files exist.